task_4.py

```diff
@@ -0,0 +1,14 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, 
+# что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# Запрашиваем и вводим количество журавликов
+s = int(input("Введите количество журавликов: "))
+
+# Находим количество журавликов, сделанных каждым ребенком
+petia_and_sereja  = s // 3
+katya = petia_and_sereja * 2  
+petia = petia_and_sereja // 2
+sereja = petia
```

Comment:
Артур не помешало бы проверить входные данные на карте с 6. Кроме того, в фаэтоне и в любом другом языке программирования не принято использовать в качестве имен переменных транслитерацию русских слов.
```suggestion
petia_and_sereja  = s // 3
katya = petia_and_sereja * 2  
petia = sereja  = petia_and_sereja // 2
```